### DISCUSSION TITLE
Remove `Ctrl-f/b` from the default keymap

### DISCUSSION THREAD

**Discussion by drybalka** [helix-editor/helix#9885] [Ideas]
The main reason for this proposal is that `Ctrl-u/d` already provide the scrolling functionality anyone would need, they are consistent across different modes, mnemonic, easy to memorize both for beginners and vim-gurus, and easily provide the same "page_up/down" motion.

The other arguments in favor of removing `Ctrl-f/b` from defaults are:
- Simplify cognitive load on a user - scrolling is always done by `Ctrl-u/d`, period.
- `Ctrl-f/b` does not work for scrolling in popups, which is confusing
- Having 2 sets of scrolling keymaps is confusing for beginners
- "page_up/down" is already mapped to physical `PageUp/Down` keys
- The default keymap is huge as it is (and quite intimidating for beginners), there is no need in making it larger
- Allow a more useful behavior for such a convenient keymaps
- I would argue, `Ctrl-f/b` is mostly used by veteran vim users, who tweak their config anyway and can easily restore the previous behavior

This issue somewhat relates to #1614

**Comment by nik-rev** [comment]
I often use `Ctrl-f/b` to scroll pages fast when I am not "looking for something", so I am opposed to this
